### PR TITLE
Add "point of order" feature to ListOfSpeakers

### PR DESCRIPTION
**Pull request by tsiegleauq**
Adds the option "point of order" to the list of speakers
- You can make a point of order even though you normally have no
permission to add yourself to the ListOfSpeakers
- You can male a point of order even though you are already on the
ListOfSpeakers (but you may only be there once)
- new points of order will be on top of the list of speakers
- Point of orders will be highlighted by a red triangle

This feature can be used to request to speak with a higher level of
urgency

### CODE REVIEW

File: server/openslides/agenda/views.py [hunk 1]
```diff
@@ -402,19 +407,17 @@ def manage_speaker(self, request, pk=None):
...
+                # Do not remove point_of_order speakers.
+
+                if not queryset.exists():
                     raise ValidationError(
-                        {"detail": "You are not on the list of speakers."}
+                        {"detail": "The user is not in the list of speakers."}
                     )
-                else:
-                    speaker.delete()
+                queryset.get().delete()
```
Comment (tsiegleauq) [inline server/openslides/agenda/views.py:420]: @FinnStutzenstein I adjusted that. The test "def test_remove_oneself(self, mock_speaker):" was failing here.

Comment (FinnStutzenstein) [reply server/openslides/agenda/views.py]: See my new commit: I change it again since with `get()` it does not work for currupted data.

**Comment by FinnStutzenstein** [comment]
@tsiegleauq I added a new commit and introduced `agenda_enable_point_of_order_speakers` in the server. If it is false, the creation of poo speakers (nice abbrev ;) ) is not allowed, but if there is one, he can speak and be removed and so on.

**Comment by FinnStutzenstein** [comment]
TODO: Check the readd-last-speaker feature. @emanuel: If the last speaker is a POO speaker, he should not be readded, should he? If so, as a normal speaker? Or should we search lor the last non-poo-speaker?

**Comment by tsiegleauq** [comment]
Tested. "Remove me" and "Remove POO" seem to work fine now.
Testes readd last speaker and it works like I would expect it to. Even though you have a POO-Speaker, readd should always recall the last speaker, which seems to work fine. Thx for taking this one over when I was away @FinnStutzenstein

**Comment by FinnStutzenstein** [comment]
@emanuelschuetze @tsiegleauq I added more logic around readd_last:

You cannot readd the last speaker, if:
- there is no last speaker (this was checked before, too)
- the last speaker is a POO speaker
- the last speaker is already waiting again.

The button in the client is also disabled, if one of these conditions matches.

**Pull request merged**